File: Day01_UnitConverter/Java/UnitConverter.java
import java.util.Scanner;

public class UnitConverter {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Unit Converter:");
        System.out.println("1. cm to inches");
        System.out.println("2. kg to pounds");
        System.out.println("3. Celsius to Fahrenheit");

        int choice = sc.nextInt();

        switch (choice) {
            case 1:
                System.out.print("Enter cm: ");
                double cm = sc.nextDouble();
                System.out.printf("%.2f cm = %.2f inches%n", cm, cm / 2.54);
                break;
            case 2:
                System.out.print("Enter kg: ");
                double kg = sc.nextDouble();
                System.out.printf("%.2f kg = %.2f pounds%n", kg, kg * 2.20462);
                break;
            case 3:
                System.out.print("Enter Celsius: ");
                double c = sc.nextDouble();
                System.out.printf("%.2f °C = %.2f °F%n", c, (c * 9/5) + 32);
                break;
            default:
                System.out.println("Invalid choice.");
        }

        sc.close();
    }
}

File: Day01_UnitConverter/CPP/unit_converter.cpp
#include <iostream>
using namespace std;

int main() {
    int choice;
    double input;

    cout << "Unit Converter:" << endl;
    cout << "1. cm to inches" << endl;
    cout << "2. kg to pounds" << endl;
    cout << "3. Celsius to Fahrenheit" << endl;
    cout << "Enter choice (1-3): ";
    cin >> choice;

    switch (choice) {
        case 1:
            cout << "Enter cm: ";
            cin >> input;
            cout << input << " cm = " << input / 2.54 << " inches" << endl;
            break;
        case 2:
            cout << "Enter kg: ";
            cin >> input;
            cout << input << " kg = " << input * 2.20462 << " pounds" << endl;
            break;
        case 3:
            cout << "Enter Celsius: ";
            cin >> input;
            cout << input << " °C = " << (input * 9 / 5) + 32 << " °F" << endl;
            break;
        default:
            cout << "Invalid choice." << endl;
    }

    return 0;
}
